examples/hubbard.jl

```diff
@@ -44,13 +44,13 @@ plot_pdos(bands; p, iatom=1, label="3D", colors=[:yellow, :orange], εrange)
 # Note that "manifold" is the standard term used in the literature for the set of atomic orbitals
 # used to compute the Hubbard correction, but it is not meant in the mathematical sense.
 U = 10u"eV"
-manifold = OrbitalManifold(atoms, Ni, "3D")
+manifold = OrbitalManifold(Ni, "3D")
```

Comment:
Add a comment that this also works with a list of integers and that this version can be used when constructing a model from an AtomsBase system.

Reply:
And mention the symbol option, too

(Many people will just look at this and never read the docs)